database/db.go

```diff
@@ -17,28 +19,50 @@ type dbConfig struct {
 	password string
 }
 
-var config = dbConfig{
-	os.Getenv("POSTGRES_HOST"),
-	os.Getenv("POSTGRES_PORT"),
-	os.Getenv("POSTGRES_USER"),
-	os.Getenv("POSTGRES_DB"),
-	os.Getenv("POSTGRES_PASSWORD"),
+func getConfig() dbConfig {
+	return dbConfig{
+		os.Getenv("POSTGRES_HOST"),
+		os.Getenv("POSTGRES_PORT"),
+		os.Getenv("POSTGRES_USER"),
+		os.Getenv("POSTGRES_DB"),
+		os.Getenv("POSTGRES_PASSWORD"),
+	}
 }
 
-var dns = fmt.Sprintf(
-	"host=%s port=%s user=%s dbname=%s password=%s sslmode=disable",
-	config.host, config.port, config.user, config.dbname, config.password)
+func getTestConfig() dbConfig {
+	return dbConfig{
+		os.Getenv("POSTGRES_HOST"),
+		os.Getenv("POSTGRES_TEST_PORT"),
+		os.Getenv("POSTGRES_USER"),
+		os.Getenv("POSTGRES_TEST_DB"),
+		os.Getenv("POSTGRES_PASSWORD"),
+	}
+}
+
+func getDns(config dbConfig) string {
+	return fmt.Sprintf(
+		"host=%s port=%s user=%s dbname=%s password=%s sslmode=disable",
```

Comment:
I get that sslmode is being disabled for testing/mvp purposes, but we should really try to figure out how to not do that.

Reply:
I created #81 to track this task.